apps/dh/api-dh/source/DataHub.WebApi.Tests/Snapshots/SchemaTests.ChangeTest.verified.graphql

```diff
@@ -708,7 +708,7 @@ input CalculationQueryInput {
 }
 
 input CancelScheduledCalculationInput {
-  guid: UUID!
+  calculationId: UUID!
```

Comment:
@ManBearTM there is a breaking change for the UI here - should we avoid it and instead keep the name as `guid`?

Reply:
No, seems like a good change 🙂